### PR TITLE
nvm 0.31.0

### PR DESCRIPTION
**Pull request by lassebm**

### CODE REVIEW

File: Library/Formula/nvm.rb [hunk 1]
```diff
@@ -31,14 +31,19 @@ def caveats; <<-EOS.undent
     #{prefix} will destroy any nvm-installed Node installations
     upon upgrade/reinstall.
 
+    If upgrading from a previous version, please remember to activate the
+    new version by restarting your shell or by executing the command below:
+
+      . $(brew --prefix nvm)/nvm.sh
```
Comment (dunn) [inline Library/Formula/nvm.rb]: This isn't Homebrew-specific information, is it?

**Comment by dunn** [comment]
The updated test is failing:
```
        Testing nvm
==> Using the sandbox
/usr/bin/sandbox-exec -f /tmp/homebrew20160224-20783-17b3rsp.sb /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Library/Homebrew -- /usr/local/Library/Homebrew/test.rb /usr/local/Library/Formula/nvm.rb --verbose
==> NODE_VERSION=stable /usr/local/Cellar/nvm/0.31.0/nvm-exec 2>&1
Error: nvm: failed
Expected "N/A: version \"stable\" is not yet installed\n" to be empty.

File: Library/Formula/nvm.rb (hunk 1 above)
Comment (lassebm) [inline Library/Formula/nvm.rb]: It is. You'll need to activate the new nvm script in the shell environment after upgrading.

Comment (lassebm) [inline Library/Formula/nvm.rb]: To be more precise, it isn't Homebrew-specific, but *also* required when installed through Homebrew :-)

Comment (dunn) [inline Library/Formula/nvm.rb]: Anything that applies generally on updates shouldn't be in caveats; in such cases we're just duplicating the upstream docs and will soon fall out-of-date.

Comment (lassebm) [inline Library/Formula/nvm.rb]: Good point, PR updated.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/b41098fe938afffa3b6a1040945cc2e4dc29ac90. Thank you for your contribution to Homebrew @lassebm; we appreciate it! :smiley_cat:

**Pull request closed without merging**